channels/pt.m3u

```diff
@@ -67,7 +67,9 @@ https://sportbar.biz/lba/123/g8/ptsport5.m3u8
 http://213.13.26.11:1935/live/santuario.stream/livestream.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/hCFVc5S.jpg" group-title="",TV Fátima
 https://streamer-b02.videos.sapo.pt/live/santuario.stream/chunklist.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/GVI8kkp.jpg" group-title="",TV Maná
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/PGWVguz.png" group-title="",TV Maná 1
+http://194.38.135.164:11000/tvmana1/alvalade/chunklist_w1384971435.m3u8
```

Comment:
Stream doesn't work (tested with VLC and streamtest)

Reply:
Link Removed